src/app/components/pool/action/add/os-picker/os-offer-tile/os-offer-tile.html

```diff
@@ -16,7 +16,7 @@
     class="select-version"
     [matMenuTriggerFor]="menu"
     matTooltip="{{'os-offer-tile.select-version-button.tooltip' | i18n}}"
-    title="{{'os-offer-tile.select-version-button.tooltip' | i18n}}">
+    title="{{'os-offer-tile.select-version-button.tooltip' | i18n}} {{name}}">
```

Comment:
A couple of minor suggestions:

- Let's use `prettyName `instead of `name `so we get "almalinux hpc" instead of "almalinux-hpc"
- Adding parentheses around the name would make it more grammatically correct. So: "Select version (almalinux hpc)" instead of "Select version almalinux hpc". Another option would be to use a template string, but parentheses work for me.